client/src/components/Navbar/Navbar.stories.js

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import Navbar from './Navbar';
+import ProfileImage from "../../assets/images/profile_large.png";
 
-storiesOf('Navbar', module)
-  .add("Default", () => (
-    <Navbar />
-  )
-)
+function handleClick() {
+  console.log("clicked");
+}
+
+storiesOf("Navbar", module).add("Default", () => (
+  <Navbar
+    profileImage={ProfileImage}
+    searchClick={handleClick}
+    messageClick={handleClick}
+    burgerClick={handleClick}
+    showCondensedMenu={true}
+    condensedMenu="Navbar__show"
+    menu2="Navbar__show"
```

Comment:
Why do we need to pass classes in like this?

Reply:
@ZakkMan I was doing this because this component didn't have state and I was setting it up so that something could be passed down to display condensed menu as well as the triangle for which page is active. If this component should have state, that functionality can be changed.

Reply:
This has been changed with the component now being a class and is no longer being passed as a prop.